pylint_odoo/checkers/modules_odoo.py

```diff
@@ -403,7 +403,8 @@ def _check_imported_packages(self, node, module_name):
 
         ext_deps = self.manifest_dict.get('external_dependencies') or {}
         py_ext_deps = ext_deps.get('python') or []
-        if module_name not in py_ext_deps and \
+        if node.as_string().split()[1].startswith('.') and \
```

Comment:
Is possible a IndexError here?

Could you run this version in all your python files locally?

Reply:
First run `pylint_odoo` over the code without this modification 
`git checkout master`

```bash
(py34-pylint20) [odoo@73f00e6f2458]~/build/Vauxoo/pylint-odoo$
 (master)$ python -m pylint -d all -e missing-manifest-dependency --load-plugins=pylint_odoo /home/odoo/build/mexico/l10n_mx_landing
No config file found, using default configuration
************* Module l10n_mx_landing.tests.test_landing
W: 10, 0: Missing external dependency "l10n_mx_base.tests.common" from manifest. More info: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies (missing-manifest-dependency)

------------------------------------------------------------------
Your code has been rated at 9.89/10 (previous run: 9.89/10, +0.00)
```

And after run `pylint_odoo` over the code with this modification 
`git checkout master-vx-fix-external-dependency-jesuszapata`

```bash
(py34-pylint20) [odoo@73f00e6f2458]~/build/Vauxoo/pylint-odoo$
 (master-vx-fix-external-dependency-jesuszapata)$ python -m pylint -d all -e missing-manifest-dependency --load-plugins=pylint_odoo /home/odoo/build/mexico/l10n_mx_landing
No config file found, using default configuration

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```